core/src/main/java/io/grpc/Status.java

```diff
@@ -238,9 +239,6 @@ public Status toStatus() {
     }
   }
 
-  private static final Charset US_ASCII = Charset.forName("US-ASCII");
```

Comment:
Just `com.google.common.collect`. And we are completely avoiding Guava (really any dependencies) for `grpc-context`, so that it is can be a more widely used component for other projects.